llvm/lib/Object/COFFImportFile.cpp

```diff
@@ -52,6 +52,32 @@ StringRef COFFImportFile::getFileFormatName() const {
   }
 }
 
+StringRef COFFImportFile::getExportName() const {
+  const coff_import_header *hdr = getCOFFImportHeader();
+  StringRef name = StringRef(Data.getBufferStart() + sizeof(*hdr));
```

Comment:
I'd prefer not to implicitly assume the MemoryBuffer is null-terminated... maybe rewrite to something like `Data.getBuffer().substr(sizeof(*hdr)).split('\0').first`.

Reply:
Good point, thanks. Fixed both comments in the new version.